.github/workflows/ci.yml

```diff
@@ -178,6 +178,7 @@ jobs:
             -DCMAKE_CXX_COMPILER=${{ matrix.cxx_compiler }} -DONEDPL_BACKEND=${{ matrix.backend }} -DONEDPL_DEVICE_TYPE=${{ matrix.device_type }} ..
           make VERBOSE=1 -j${BUILD_CONCURRENCY} ${make_targets}
           ctest --timeout ${TEST_TIMEOUT} --output-on-failure ${ctest_flags}
+          exit 1
```

Comment:
Let's remove it, because it does not solve the initial problem. It forces "red" status for all the runs, even if they finish without issues.